### PR TITLE
Set up Travis CI

### LINKED ISSUE #28: Travis CI!
- [ ] Add `.travis.yml`
- [ ] Badges!

### PR DESCRIPTION
**Pull request by sharplet**
- [x] Add `.travis.yml`
- [x] Badges!

Fixes #28.

### CODE REVIEW

**Comment by robrix** [comment]
(You can use `Fixes #28` in the PR description to link the issue and PR to one another, incidentally. It will also result in closing out the issue when the PR gets merged, which is nice.)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: swift
```
Comment (robrix) [inline .travis.yml]: Data point: Carthage uses [`objective-c` for this](https://github.com/Carthage/Carthage/blob/master/.travis.yml#L1).

**Comment by sharplet** [comment]
Thanks, good tip!

File: .travis.yml (hunk 1 above)
Comment (sharplet) [inline .travis.yml]: Yeah I was punting to see if `swift` was supported as a languagage identifier. Turns out, no ;) https://travis-ci.org/robrix/Set/builds/47711419

**Comment by sharplet** [comment]
FYI I'm planning to push whatever rubbishy commits I need to on this branch while I'm debugging the setup, and I rebase to tidy up once it's done.

**Comment by sharplet** [comment]
[Yay](https://travis-ci.org/robrix/Set/builds/47716584)! So I'd normally create a `Rakefile` for this kind of thing, do you have any preference @robrix? Or we could potentially even just leave it directly in `.travis.yml` as it's a very simple project.

**Comment by robrix** [comment]
> rebase to tidy up once it's done

If you like. I’m not fussed about that, personally.

**Comment by robrix** [comment]
I dig leaving it in the `.travis.yml` like it is right now. Pretty nice having it so lean!

I didn’t know about `xcpretty`, that’s nice too :+1:

If you want to rebase, feel free, otherwise I’ll just merge this because everything is wonderful :sparkles:

**Comment by sharplet** [comment]
:shipit:

**Comment by sharplet** [comment]
Forget about the badge, I'll quickly add that

**Comment by sharplet** [comment]
All done sir.

**Comment by robrix** [comment]
:sparkles: excellent, thank you so much!

**Comment by sharplet** [comment]
You're welcome! 😃

**Pull request merged**